tensorboard/components/vz_line_chart2/microbenchmark/reporter.ts

```diff
@@ -0,0 +1,169 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {Result} from './types';
+
+export function htmlTableReporter(results: Result[]) {
+  const displayResults = results
+    .map(
+      ({
+        totalTimeInMs,
+        numIterations,
+        timePerIterationInMs,
+        timeInGcInMs,
+        benchmark,
+      }) => {
+        const avgTimeInMs = totalTimeInMs / numIterations;
+        const variance =
+          timePerIterationInMs
+            .map((time) => {
+              const diff = time - avgTimeInMs;
+              return diff * diff;
+            })
+            .reduce((sigma, diff) => {
+              return sigma + diff;
+            }, 0) / numIterations;
+        const avgTimeInGcInMs = timeInGcInMs
+          ? timeInGcInMs.reduce((a, b) => a + b, 0) / numIterations
+          : null;
+
+        return {
+          name: benchmark.name,
+          min: Math.min(...timePerIterationInMs),
+          max: Math.max(...timePerIterationInMs),
+          numIterations,
+          variance,
+          avgTimeInMs,
+          avgTimeInGcInMs,
+        };
+      }
+    )
+    .map(
+      ({
+        name,
+        min,
+        max,
+        numIterations,
+        variance,
+        avgTimeInMs,
+        avgTimeInGcInMs,
+      }) => {
+        return {
+          name,
+          numIterations,
+          avgTime: `${avgTimeInMs.toFixed(3)}ms / run`,
+          min: `${min.toFixed(3)}ms`,
+          max: `${max.toFixed(3)}ms`,
+          stdDeviation: Math.sqrt(variance).toFixed(6),
```

Comment:
1. stdDeviation of what? Of time? Rename variable to make it clearer.
2. Since you suffix the avg key name with units (e.g., `InMs`) here, you might as well do it for stdDeviation too.